test/e2e/auth/certificates.go

```diff
@@ -258,7 +260,9 @@ var _ = SIGDescribe("Certificates API [Privileged:ClusterAdmin]", func() {
 					break
 				}
 			}
-			framework.ExpectEqual(found, true, fmt.Sprintf("expected certificates API version, got %#v", group.Versions))
```

Comment:
It's worth calling out that `fmt.Sprintf` by itself also was a bug by itself here: `ExpectEqual` takes a format string plus arguments, so `fmt.Sprintf` was at best useless. But it could also have produced a string with `%` specifiers, which then would have been expanded again, which would have failed.